prediction_market_agent/agents/gptr_agent/deploy.py

```
@@ -8,18 +8,25 @@
     BinaryKellyBettingStrategy,
 )
+from prediction_market_agent_tooling.deploy.betting_strategy import (
+    BettingStrategy,
+    BinaryKellyBettingStrategy,
+)
 from prediction_market_agent_tooling.deploy.trade_interval import (
     FixedInterval,
     TradeInterval,
 )
 from prediction_market_agent_tooling.gtypes import USD
+from prediction_market_agent_tooling.gtypes import USD
```

Ruff (0.12.2)

Redefinition of unused `USD` from line 19

Remove definition: `USD`

(F811)